Classes/ComPspdfkitView.m

```diff
@@ -51,7 +51,7 @@ - (void)createControllerProxy {
 
             // Support for tinting the navigation controller/bar
             if (options[@"barColor"]) {
-                UIColor *barColor = [[TiColor colorNamed:options[@"barColor"]] color];
+                UIColor *barColor = (UIColor *)[[TiColor colorNamed:options[@"barColor"]] color];
```

Comment:
This should return a `UIColor` instance already. If it returns `id`, use the `_color` getter.

Reply:
Thanks @hansemannn 👍